src/labeler.spec.ts

```diff
@@ -0,0 +1,130 @@
+const {getFileLabel, fileStatus, labelType} = require('./labeler')
+
+describe('getFileLabel', () => {
+  describe('when a main page is added', () => {
+    it('should return new command label', () => {
+      expect(getFileLabel({
+        filename: 'pages/common/cat.md',
+        status: fileStatus.added,
+      })).toBe(labelType.newCommand);
+    });
+  });
+
+  describe('when a main page is modified', () => {
+    it('should return page edit label', () => {
+      expect(getFileLabel({
+        filename: 'pages/windows/dir.md',
+        status: fileStatus.modified,
+      })).toBe(labelType.pageEdit);
+    });
+  });
+
+  describe('when a main page is removed', () => {
+    it('should return page edit label', () => {
+      expect(getFileLabel({
+        filename: 'pages/osx/du.md',
+        status: fileStatus.modified,
+      })).toBe(labelType.pageEdit);
+    });
+  });
+
+  describe('when a main page is renamed', () => {
+    it('should return page edit label', () => {
+      expect(getFileLabel({
+        previous_filename: 'pages/linux/ls.md',
+        filename: 'pages/common/ls.md',
+        status: fileStatus.renamed,
+      })).toBe(labelType.pageEdit);
+    });
+  });
+
+  describe('when a translation page is added', () => {
+    it('should return translation label', () => {
+      expect(getFileLabel({
+        filename: 'pages.de/common/git.md',
+        status: fileStatus.modified,
+      })).toBe(labelType.translation);
+    });
+  });
+
+  describe('when a translation page is modified', () => {
+    it('should return translation label', () => {
+      expect(getFileLabel({
+        filename: 'pages.es/common/7z.md',
+        status: fileStatus.modified,
+      })).toBe(labelType.translation);
+    });
+  });
+
+  describe('when a translation page is removed', () => {
+    it('should return translation label', () => {
+      expect(getFileLabel({
+        filename: 'pages.pt_BR/common/mkfs.btrfs.md',
+        status: fileStatus.modified,
```

Comment:
Need to double check these.

```suggestion
        status: fileStatus.removed,
```